### PR TITLE
Update issue templates

### LINKED ISSUE #74: Adding template for issues and pull requests
To achieve a higher quality in the issues and in the pull requests we might consider adding templates for both, the issues and the pull requests. The templates are quite easy to create, but what should go in there?
- issues: What kind of issue is committed and what is the relevant information that should be provided by the issue.  It could also include a hint to stick to the code of conduct. 
-  pull requests: Adding a checklist, if everything is complete and all required information is contained.

### PR DESCRIPTION
**Pull request by anilturkmayali**
Started with an issue template for requests to change the content. (This includes all types of changes such as deletion, adjustment, enhancement, replacement, etc.).

Closes #74

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/content-change-request.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  on any existing part of the documentation
+title: Content Change Request
+labels: ''
+assignees: ''
+
+---
+
+**Which part of the repository is your change request related to?**
+Please provide the exact URL(s) of the page(s) [...]
+
```
Comment (ssteinbuss) [inline .github/ISSUE_TEMPLATE/content-change-request.md:13]: I would add an additional point here. 
** Please describe  which content you would expect here **
Describe a little more in detail which information or resources you would expect here.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
```
Comment (ssteinbuss) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]: I would add here one additional point:
 ** Please describe the problem that you would like to solve with this feature **
Which problem do you want to have solved and how is this related to IDS and the work of IDSA.

**Review by ssteinbuss — CHANGES_REQUESTED** [review]
- Please add the mentioned questions to the templates. 
-  Check the formatting of the templates. The line breaks do not really fit. Please check the rendered markdown.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (anilturkmayali) [reply .github/ISSUE_TEMPLATE/feature_request.md:12]: Thank you for the comment. I replaced the first question with this one, to refrain from having two similar questions.

**Review by anilturkmayali — COMMENTED** [review]
Applied the feedback and made the corrections on questions and on formatting.
